api/utils/fetch.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpMethod, FetchOptions } from 'common/types/requests';
```

Comment:
Not sure that this file is needed, because i wrote it for external ettu API (so there is CORS wrapper that we probably don't need for map). And maybe there will be other API response format

Reply:
Yep, we already have our own API without problems with CORS. Remove it